cert-expiry-finder

```diff
@@ -296,18 +296,20 @@ expiry_after() {
 
 list_k8s_cert_expiry_days() {
     local name namespace notafter unixtime delta context
-    if ! which kubectl; then
+    if ! command -v kubectl 2>/dev/null; then
```

Comment:
Oops. Should've been `>`, not `2>`. My bad.

Reply:
changed